### PR TITLE
feat: add orbiting circles for community section on landing page

### LINKED ISSUE #2159: [feature]: add orbiting circles for community section on landing page
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

Idea is to add Orbiting circles animation like https://magicui.design/docs/components/orbiting-circles to the community section of keploy landing page with socials card. 

### Why should this be worked on?

better design

### Repository

website

### PR DESCRIPTION
**Pull request by techmannih**
close https://github.com/keploy/keploy/issues/2159
Issue https://github.com/keploy/keploy/issues/2159

@khareyash05 @Swpn0neel please review PR and merge it.

Now 

https://github.com/user-attachments/assets/4c964d85-ef01-481f-91de-a7d485c988bc

### CODE REVIEW

**Comment by Neeraj-gagat** [comment]
Duplicate #87 
Bro you just copy pasted exact same changes I did. Exact word to word

**Comment by Neeraj-gagat** [comment]
You just copy pasted exact same changes I did.

File: components/community.tsx [hunk 1]
```diff
@@ -59,7 +63,7 @@ function SocialLinkCard({
       rel="noopener noreferrer"
       style={showExtraStyle ? { ...style } : {}}
       className={`${
-        showExtraStyle ? "circle" : "w-3/4 mx-auto mb-5"
+        showExtraStyle ? "circle" : "w-[280px] mx-auto mb-5"
```
Comment (Sonichigo) [inline components/community.tsx]: any specific for the 280px ?

**Comment by techmannih** [comment]
@khareyash05 please review

**Review by khareyash05 — CHANGES_REQUESTED** [review]
Could you please address the above comment. Thanks

**Comment by techmannih** [comment]
> You just copy pasted exact same changes I did.

please check once it's totally changed, @khareyash05 also check please

**Comment by khareyash05** [comment]
Closing due to unaddressed comments

**Comment by techmannih** [comment]
@khareyash05  Which comment did I not reply to? I am working on the issue. You can also check that this is not copied code.

**Pull request closed without merging**